examples/storage/main.tf

```
@@ -1,21 +1,68 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 provider "google" {
-  credentials = "${file(var.credentials_path)}"
+  version = "~> 2.15.0"
+}
+
+resource "random_string" "suffix" {
+  length  = 4
+  special = "false"
+  upper   = "false"
+}
+
+locals {
+  storage = {
+    name    = "my_storage_${random_string.suffix.result}"
+    project = var.project_id
+  }
+}
+
+resource "google_compute_network" "vpc_network" {
+  project                 = var.project_id
+  name                    = "example-network-${random_string.suffix.result}"
+  auto_create_subnetworks = "true"
 }
 
 module "gsuite-export" {
```

Modules should be delimited with underscores.

```suggestion
module "gsuite_export"
```